src/services/stringCompletions.ts

```diff
@@ -565,8 +565,17 @@ function getCompletionEntriesForRelativeModules(literalValue: string, scriptDire
     }
 }
 
-function getSupportedExtensionsForModuleResolution(compilerOptions: CompilerOptions): readonly Extension[][] {
-    const extensions = getSupportedExtensions(compilerOptions);
+function getSupportedExtensionsForModuleResolution(compilerOptions: CompilerOptions, typeChecker?: TypeChecker): readonly Extension[][] {

```

Comment:
Oops, one thing I didn’t notice in my review: this should probably be `string[][]` and drop the cast below. It will certainly have to be plumbed through to other functions, but hopefully shouldn’t be too bad. Do you mind giving that a shot in a follow-up PR?